src/modules/home/components/LogoutButton.js

```diff
@@ -6,19 +6,12 @@ import { homePath } from '../../home/routes/HomeRoute'
 
 export const LogoutButton = () => {
     const navigate = useNavigate()
-    const { setUser } = useContext(UserContext)
+    const { logout } = useContext(UserContext)
 
     const onClick = () => {
-        localStorage.clear()
-        const user = {
-            firstName: '',
-            lastName: '',
-            username: '',
-            email: '',
-            token: ''
-        }
-        setUser(user)
+        logout()
         navigate(homePath)
+        localStorage.clear()
```

Comment:
this should move to the logout method as every single time logout is called it should happen, saves us from doing it in every componenet we happened to call logout method in

Reply:
done